### PR TITLE
Add more documentation to hydro::Tags.

### PR DESCRIPTION
**Pull request by markscheel**
## Proposed changes

Fix documentation bugs and add more documentation to some of the hydro tags.

### Types of changes:

- [x] Bugfix
- [x] New feature

### Component:

- [x] Documentation

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/PointwiseFunctions/Hydro/Tags.hpp [hunk 1]
```diff
@@ -94,7 +100,8 @@ struct MagneticFieldDotSpatialVelocity : db::SimpleTag {
   }
 };
 
-/// The oneform of the magnetic field.
+/// The oneform of the magnetic field.  Note that \f$B^i\f$ is raised
```
Comment (fmahebert) [inline src/PointwiseFunctions/Hydro/Tags.hpp]: For consistency: use "one-form" here too

**Review by nilsdeppe — DISMISSED** [review]
I have no additional comments. I verified it renders correctly.

**Comment by kidder** [comment]
I have no additional comments either

**Comment by kidder** [comment]
@markscheel please fix the one change and squash

**Review by kidder — APPROVED** [review]

**Comment by kidder** [comment]
@fmahebert please look at

**Review by fmahebert — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**